### DISCUSSION TITLE
Why is my Nuxt project taking so much time to start and update in dev server ?

### DISCUSSION THREAD

**Discussion by icepick4** [nuxt/nuxt#25723] [Questions]
Hello,

I'm currently working on a Nuxt project and have encountered an issue. When I start the development server using npm run dev, the initial launch and subsequent rebuilds whenever I make local changes are taking a very long time (> 1 min to start & ~ 10 sec when saving)

### Details about my setup : 

------------------------------
- Operating System: Windows_NT
- Node Version:     v20.11.0
- Nuxt Version:     3.10.0
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  npm@10.2.4
- Builder:          -
- User Config:      devtools, modules, runtimeConfig, auth, mongoose, postcss, css, ui, experimental
- Runtime Modules:  @nuxt/ui@2.13.0, @sidebase/nuxt-auth@0.6.7, @nuxt/image@1.3.0, nuxt-mongoose@1.0.5
- Build Modules:    -
------------------------------

Here is the launch of the dev server : 

![image](https://github.com/nuxt/nuxt/assets/82316285/beda8081-391c-46cf-95a3-544ac5e8835e)

Here is my `nuxt.config.ts` : 

```ts
// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
    devtools: { enabled: true },
    modules: [
        '@nuxt/ui',
        '@sidebase/nuxt-auth',
        '@nuxt/image',
        'nuxt-mongoose'
    ],
    runtimeConfig: {
        authSecret: process.env.AUTH_SECRET,
        public: {
            stripeKey: process.env.STRIPE_KEY
        }
    },
    auth: {
        baseURL: process.env.AUTH_ORIGIN,
        provider: {
            type: 'authjs'
        }
    },
    mongoose: {
        uri: process.env.MONGODB_URL,
        options: {},
        modelsDir: 'models'
    },
    postcss: {
        plugins: {
            tailwindcss: {},
            autoprefixer: {}
        }
    },
    css: ['~/assets/css/main.css'],
    ui: {
        safelistColors: ['blue']
    }
});
```

### What I tried 

I read that maybe it was because of localhost, so I switched to 127.0.0.1 instead, it did not changed that much.

**Comment by TheAlexLichter** [comment]
Share your tailwind config please 👀 
Also, why do you add tailwind "manually" even though provided through Nuxt UI?

**Comment by icepick4** [comment]
My tailwind config : 

```js
/** @type {import('tailwindcss').Config} */
const colors = require('tailwindcss/colors');
export default {
    content: [
        './components/**/*.{js,vue,ts}',
        './layouts/**/*.vue',
        './pages/**/*.vue',
        './plugins/**/*.{js,ts}',
        './app.vue',
        './error.vue'
    ],
    theme: {
        extend: {
            colors: {
                gray: colors.neutral,
                red: colors.red,
                yellow: colors.amber,
                blue: colors.blue,
                purple: colors.violet,
                green: colors.green,
                orange: colors.orange,
                cyan: colors.cyan,
                pink: colors.pink,
                white: colors.white,
                black: colors.black
            },
            screens: {
                xs: '500px',
                sm: '640px',
                md: '768px',
                lg: '1024px',
                xl: '1280px',
                xxl: '1536px',
                '2xl': '1536px'
            },
            fontFamily: {
                title: "'title'"
            }
        }
    },
    plugins: []
};
```

To add tailwind I just followed [this tutorial](https://tailwindcss.com/docs/guides/nuxtjs) from the official doc of tailwind

I should cancel that and remove these packages : tailwindcss postcss autoprefixer ?

**Comment by icepick4** [comment]
Thank you so much it works ! I just removed the 3 packages (tailwindcss postcss autoprefixer) and changed my config to this : 

```js
// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
    devtools: { enabled: true },
    modules: [
        '@nuxt/ui',
        '@sidebase/nuxt-auth',
        '@nuxt/image',
        'nuxt-mongoose'
    ],
    runtimeConfig: {
        authSecret: process.env.AUTH_SECRET,
        public: {
            stripeKey: process.env.STRIPE_KEY
        }
    },
    auth: {
        baseURL: process.env.AUTH_ORIGIN,
        provider: {
            type: 'authjs'
        }
    },
    mongoose: {
        uri: process.env.MONGODB_URL,
        options: {},
        modelsDir: 'models'
    },
    ui: {
        safelistColors: ['blue']
    },
    tailwindcss: {
        configPath: 'tailwind.config.js',
        cssPath: '~/assets/css/main.css'
    }
});
```

And now it starts in 10 secondes and it refreshes in 1/2 seconds ! 
The problem was that I had two occurence of tailwind I think.
